pkgs/servers/sql/cockroachdb/cockroachdb-bin.nix

```diff
@@ -6,21 +6,25 @@
 
 let
   version = "23.1.7";
-  name = "cockroachdb-${version}";
+  name = "cockroachdb";
```

Comment:
@wegank Why did you drop version from the name?

Reply:
It doesn't seem very intuitive to me that the program name should contain the version. Would you like me to revert the change?